pkg/kotsadmsnapshot/backup.go

```diff
@@ -930,3 +946,59 @@ func prepareIncludedNamespaces(namespaces []string) []string {
 	}
 	return includedNamespaces
 }
+
+// excludeShutdownPodsFromBackup will exclude pods that are in a shutdown state from the backup
+// this is to prevent the hook backup from failing if a pod is in a shutdown state and cannot be backed up
+func excludeShutdownPodsFromBackup(ctx context.Context, clientset kubernetes.Interface, backup *velerov1.Backup) (err error) {
+	namespaces := backup.Spec.IncludedNamespaces
+	if len(namespaces) == 0 {
+		return
+	}
+
+	// if the backup includes all namespaces, we need to get the list of namespaces
+	if namespaces[0] == "*" {
```

Comment:
`*` is not necessarily the first element in the array. i would suggest looping through the array to detect it.
```suggestion
	if namespaces[0] == "*" {
```

Reply:
hmm, actually nevermind, i see that the `prepareIncludedNamespaces` already handles that